ui/ui-frontend/projects/collect/src/app/collect/archive-search-collect/archive-search-criteria/components/criteria-search/criteria-search.component.html

```diff
@@ -15,7 +15,7 @@
                   ? (criteriaValue.value.value | date: 'dd/MM/yyyy')
                   : criteriaValue.valueTranslated
                     ? ('COLLECT.SEARCH_CRITERIA_FILTER.' + getCategoryName(criteriaVal.category) + '.' + criteriaValue.label | translate)
-                    : criteriaValue.label
+                    : (getCriteriaLabel(criteriaVal.key, criteriaValue) | truncate: 50)
```

Comment:
Dans le cas DATE, on n'utilise pas getCriteriaLabel sur les autres branches

```suggestion
                    : criteriaValue.label
```